apps/prairielearn/elements/pl-code/pl-code.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, **options: Any) -> None:
         pygments.lexer.Lexer.__init__(self, **options)
         self.compress = options.get("compress", "")
 
-    def get_tokens_unprocessed(self, text: str) -> list[tuple[int, Type, str]]:
+    def get_tokens_unprocessed(self, text: str) -> list[tuple[int, _TokenType, str]]:
```

Comment:
The previous form was failing type checking with the following:

```
/Users/nathan/git/PrairieLearn/apps/prairielearn/elements/pl-code/pl-code.py
  /Users/nathan/git/PrairieLearn/apps/prairielearn/elements/pl-code/pl-code.py:61:21 - error: Expression of type "list[tuple[Literal[0], _TokenType, str]]" cannot be assigned to return type "list[tuple[int, Type[Unknown], str]]"
    "_TokenType" is incompatible with "Type[Unknown]" (reportGeneralTypeIssues)
```

I couldn't immediately trace this back to anything specific in the changelog.